components/TrackingSection/index.js

```diff
@@ -0,0 +1,3 @@
+export default function TrackingSection() {
+  return <p>TrackingSection</p>;
+}
```

Comment:
Why do you need a component with only this content? Do you use it more often?

Besides, I would argue that a `<p>´ tag is not the correct semantic for one word. It is rather a `<span>` or is it a heading instead?

Reply:
It is only meant to be a placeholder as creating the page is task of my next user story. I started with adding the page-file, then got the hint to first refactor my code and that is why the component is still (almost) empty. Sorry for the misunderstanding.